webapp/src/share.tsx

```diff
@@ -195,9 +195,7 @@ export class ShareEditor extends auth.Component<ShareEditorProps, ShareEditorSta
         const { visible, projectName: newProjectName, title, screenshotUri } = this.state;
         const { simScreenshot, simGif } = pxt.appTarget.appTheme;
         const hasIdentity = auth.hasIdentity() && this.isLoggedIn();
-        const light = !!pxt.options.light;
-        const thumbnails = pxt.appTarget.cloud && pxt.appTarget.cloud.thumbnails
```

Comment:
Was `pxt.appTarget.cloud.thumbnails` superfluous in some way? I don't really know what it represents, but was curious as to why it was removed.

Reply:
I'm not sure as to what it did, honestly. But the `light` and `thumbnails` variables were never used, and for the case of share dialog, if it's specified in `pxtarget.json` that we do or don't want the thumbnails,  I think that's enough to either supply or ignore them.

Reply:
Ah, got it. I missed that it was unused before your change.